python_modules/dagster/dagster/core/events.py

```diff
@@ -283,7 +276,7 @@ def logger_to_kwargs(logger_message):
 
     event_cls = EVENT_CLS_LOOKUP[event_type]
     if issubclass(event_cls, PipelineEventRecord):
-        return merge_two_dicts(base_args, {'pipeline_name': logger_message.meta['pipeline']})
+        return merge_dicts(base_args, {'pipeline_name': logger_message.meta['pipeline']})
```

Comment:
I guess the type check is nice, but fwiw otherwise this is the same as just writing `dict(base_args, **{'pipeline_name': logger_message.meta['pipeline']})` or `dict(base_args, pipeline_name=logger_message.meta['pipeline'])`.

Reply:
ah yes in this case that is nicer